public/public.php

```diff
@@ -1054,7 +1047,7 @@ public function render_caption_string( int $id, array $data = [], array $args =
 		}
 
 		// don’t render the caption for own images if the admin choose not to do so
-		if ( self::is_standard_source( 'exclude' ) && ISC_Public::use_standard_source( $id ) ) {
+		if ( Standard_Source::hide_standard_source_for_image( $id ) ) {
 			ISC_Log::log( sprintf( 'render_caption_string() skipped overlay for "own" image ID "%s"', $id ) );
 			return '';
 		}
```

Comment:
**Suggestion:** It seems that you are using the `Standard_Source` class methods statically. If `Standard_Source` is not designed as a utility class, consider instantiating it and using its methods as instance methods. This can make your code more testable and maintainable. [best practice]
```suggestion
  $standardSource = new Standard_Source();
  if ( $standardSource->hide_standard_source_for_image( $id ) ) {
      ISC_Log::log( sprintf( 'render_caption_string() skipped overlay for "own" image ID "%s"', $id ) );
      return '';
  }
```